apps/rule-manager/client/src/ts/components/RulesTable.tsx

```diff
@@ -238,6 +243,7 @@ const RulesTable = () => {
       <EuiFlexItem grow={2}>
         {rules &&
           <EuiInMemoryTable
+            rowProps={getRowProps}
             loading={isLoading}
```

Comment:
```suggestion
	css={css`.euiTableRow.euiTableRow-isSelected { background-color: rgba(0, 119, 204, 0.1); }`}
        loading={isLoading}
```

Reply:
If we want a slightly more prominent selection colour

Reply:
This is perfect to my eyes, nice!